plugins/ps/pre-deploy

```diff
@@ -5,4 +5,5 @@ source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
 APP="$1"
 
+get_procfile $APP
```

Comment:
That said, if you'd prefer I can revert `generate_scale_file` changes and only do this for dockerfile apps, but it would mean double extraction for dockerfile apps to generate the scale file and save the procfile for use by `dokku deploy`.